docs/_layouts/pages.html

```diff
@@ -2,25 +2,106 @@
 layout: default
 class: fill-light
 options: full
+examples: 
+  - name: Getting started
+    id: get-started
+    level: h3
+  - name: Add a style
+    id: start-styles
+    level: h4
+    class: icon paint quiet
+  - name: Add a layer
+    id: start-data
+    level: h4
+    class: icon tileset quiet
+  - name: Dynamic Styling
+    id: data-visualization
+    level: h3
+  - name: Style points
+    id: point
+    level: h4
+    class: icon marker quiet
+  - name: Style lines
+    id: line
+    level: h4
+    class: icon line quiet
+  - name: Style polygons
+    id: polygon
+    level: h4
+    class: icon polygon quiet 
+  - name: User Interaction
+    id: user-interaction
+    level: h3
+  - name: Popups
+    id: popup
+    level: h4
+    class: icon contact quiet
+  - name: Filter
+    id: filter
+    level: h4
+    class: icon layers quiet
+  - name: Animations
+    id: animation
+    level: h3
+  - name: Plugins
+    level: h3
+  - name: Geocoding
+    id: mapbox-geocoding
+    level: h4
+    class: icon search quiet
+  - name: Directions
+    id: mapbox-directions
+    level: h4
+    class: icon street quiet
+  - name: GL Draw
+    id: gl-draw
+    level: h4
+    class: icon pencil quiet
+  - name: Change map options
+    id: change-option
+    level: h3
+  - name: Misc
+    id: misc
+    level: h3
 ---
 
 <link href='{{site.url}}/css/docs.css' rel='stylesheet' />
 <link href='{{site.baseurl}}/site.css' rel='stylesheet' />
 <script src='{{site.baseurl}}/js/site.js'></script>
 
+<style>
+  h3 {
+    padding-top:15px;
+  }
+
+  h4 {
+    font-weight: normal;
+  }
+
+  .display-none {
+    display: none;
+  }
+
+</style>
+
 <!--Fixed sidebar navigation-->
 <div class='docnav hide-mobile'>
   <div class='limiter'>
     <div class='col3 contain'>
       <nav class='scroll-styled quiet-scroll small'>
         {% if page.url contains 'example' %}
-          <div class='space-bottom'>
-            <a class='block quiet active strong small truncate' href='{{site.baseurl}}/examples/'>Examples</a>
+          <div>
+            <input id='filter-input' type='text' name='filter' placeholder='Filter examples' />
           </div>
-          {% for eg in site.categories.example reversed %}
-            <a href='{{site.baseurl}}{{eg.url}}' class='block small truncate {% if eg.title == page.title %} active{% endif %}'>{{eg.title}}</a>
-          {% endfor %}
-        {% endif %}
+          {% for example in layout.examples %}
+          <div class='space-bottom1'>
+            <{{example.level}} class='heading {{example.class}}'>{{example.name}}</{{example.level}}>
+            {% for listing in site.tags[example.id] order:ascending %}
```

Comment:
This line is triggering build errors for me

```
Liquid Warning: Liquid syntax error (line 32): Invalid attribute in for loop. Valid attributes are limit and offset in "listing in site.tags[example.id] order:ascending" in /_layouts/pages.html
```

The [Liquid docs](https://help.shopify.com/themes/liquid/tags/iteration-tags) do not list `order` as a valid attribute